package.json

```
@@ -3,37 +3,54 @@
   "displayName": "Status Bar Env Var",
   "description": "Display an environment variable value in the status bar.",
   "license": "MIT",
-  "version": "0.1.2",
+  "version": "0.2.0",
   "author": "Leigh McCulloch",
   "publisher": "leighmcculloch",
+  "contributors": [
+    "Seppe Stas"
+  ],
   "repository": {
     "type": "git",
     "url": "https://github.com/leighmcculloch/vscode-extension-status-bar-env-var"
   },
   "icon": "resources/icon.png",
   "engines": {
-    "vscode": "^1.55.0"
+    "vscode": "^1.65.0"
   },
-  "extensionKind": ["ui"],
+  "extensionKind": [
+    "ui"
+  ],
   "categories": [
     "Other"
   ],
   "activationEvents": [
-    "*"
+    "onStartupFinished"
   ],
   "contributes": {
     "configuration": [
       {
         "title": "Status Bar Env Var",
         "properties": {
           "statusBarEnvVar.environmentVariableNames": {
-            "type": "string",
-            "default": "DOCKER_HOST",
-            "description": "A comma-separated list of environment variables to display in the status bar.",
-            "scope": "application"
+            "type": "array",
+            "scope": "window",
```

Would changing this back to `application` cause it to apply to VSCode and its extensions? What does `window` do here?
```suggestion
            "scope": "application",
```

---

If I remember correctly, application means it is an application only level config, available only as a global user setting, meaning it is the same for all instances of VSCode. No per workspace configuration is provided.

With window, you can provide the configuration to both the global user config and workspace level configs. E.g you can configure variables that are the same for all instances of VSCode, but also provide a options per workspace.

Other options are workspace, only allowing config on a per workspace basis, project, which allows different configs per project in the workspace, ...

For me window makes sense because the status bar is also window "scoped". It provides the flexibility I need while being backwards compatible with the application level config, without being confusing like e.g project level config would be.

I think a good idea would be to decorate the task bar items with icons depending on the scope and/or adding some text to the tooltip.